src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/NonEmptyAtclauseDescriptionCheck.java

```diff
@@ -54,6 +54,11 @@
     }
 
     @Override
+    public int[] getRequiredJavadocTokens() {
+        return getDefaultJavadocTokens();
+    }
```

Comment:
hmm, the whole reason of the issue was to customize NonEmptyAtclauseDescription , but with such method we do not allow customization. 
required should be empty array.